### PR TITLE
Support log follow with podman - closes #112

### LINKED ISSUE #112: Suport logs --follow with podman runtime
Currently `/rest/v1/logs` endpoint works, but `--follow` parameter isn't handled for podman runtime.

To run itzo with podman, you should run in GCE and  add this to your provider config:
```
bootImageSpec:
    project: "elotl-kip"
    image: "elotl-kipdev-podman"
```
and
```yaml
cellConfig:
    itzoFlag-use-podman: true
```
you can also write e2e tests and run them locally - check [server_podman_test.go](https://github.com/elotl/itzo/blob/master/pkg/server/server_podman_test.go)

as `logsHandler` is pretty long and verbose function, we should at least try to refactor it ;)

### PR DESCRIPTION
**Pull request by hidalgopl**

### CODE REVIEW

**Comment by ldx** [comment]
Is this ready for review?

**Comment by hidalgopl** [comment]
> Is this ready for review?

yeap!

File: pkg/runtime/podman.go [hunk 1]
```diff
@@ -172,7 +168,11 @@ func (pcs *PodmanContainerService) CreateContainer(unit api.Unit, spec *api.PodS
 				break
 			}
 		}
-		path := volume.HostPath.Path
+		path := filepath.Join("/tmp/itzo/packages", volume.Name)
```
Comment (ldx) [inline pkg/runtime/podman.go:171]: Shouldn't we check here if `volume.PackagePath` is non-nil?

**Review by ldx — APPROVED** [review]
LGTM :+1:

**Review by jrroman — APPROVED** [review]
👍🏼 

There is one merge conflict that needs to be fixed just in case you didn't realize

**Pull request merged**